wikimedia/wikipedia-ios
# Add highlighted/selected cell state to Notifications Center cells

**Phabricator:** https://phabricator.wikimedia.org/T297977

### Notes
Adds a background fill that resembles the standard highlighted/selected cell state to Notifications Center cells. This was a little more straightforward than I was expecting it to be (because of our custom swipe actions), but I could be missing something.

### Test Steps
1. Tap and hold on a Notifications Center cell
2. Confirm it shows a background highlighted fill color (blue)
3. Switch to Edit mode
4. Tap or hold a cell and confirm it shows with a background highlighted fill color
5. Switch themes and repeat

### Screenshots/Videos
From Figma:
![figma_selection](https://user-images.githubusercontent.com/5652327/152602826-96a68b94-8a6f-4631-b428-b82f7a3d2d8e.png)
